pkg/api/handlers/auditlogexport.go

```diff
@@ -0,0 +1,452 @@
+package handlers
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/gptscript-ai/go-gptscript"
+	"github.com/obot-platform/obot/apiclient/types"
+	"github.com/obot-platform/obot/pkg/api"
+	"github.com/obot-platform/obot/pkg/auditlogexport"
+	v1 "github.com/obot-platform/obot/pkg/storage/apis/obot.obot.ai/v1"
+	"github.com/obot-platform/obot/pkg/system"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apiserver/pkg/server/options/encryptionconfig"
+	kclient "sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+type AuditLogExportHandler struct {
+	client           kclient.Client
+	gptClient        *gptscript.GPTScript
```

Comment:
The `api.Context` object has these already. We should remove them from this handler and use the one there instead. It helps with managing dependency cycles in the future.